scripts/wheel/patch_wheel.py

```
@@ -40,23 +40,38 @@ def process(out_dir: Path, whl: Path):
         py_version = "3.10"
     elif "cp311" in str(whl):
         py_version = "3.11"
-    else:
+    elif "cp312" in str(whl):
         py_version = "3.12"
+    elif "cp313" in str(whl):
+        py_version = "3.13"
+    elif "py3-none":
+        py_version = None
+    else:
+        assert False, f"Unknow python version in {whl}"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)